evalml/tests/component_tests/test_column_selector_transformers.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 import pandas as pd
 import pytest
+from pandas.testing import assert_frame_equal
```

Comment:
In general, updated to import rather than calling pandas.testing. assert_frame_equal everywhere. Makes those lines much shorter and readable IMO :D